### PR TITLE
Update text for Kirsten Jones 

### LINKED ISSUE #436: Kirsten Update
Update text about Kirsten at her request

### PR DESCRIPTION
**Pull request by nikeshashar**
Closes #436

### CODE REVIEW

**Comment by spike01** [comment]
:shipit:

File: data/graduates.yml [hunk 1]
```diff
@@ -46,17 +46,17 @@
...
-      answer: Sometimes it felt like there was a lot you had to learn and it was difficult to always keep on top of everything knowing there was something new coming new week. It was challenging getting used to that feeling of not finishing things and feeling okay to move onto the next thing.
+      answer: There is a lot to learn and it was sometimes difficult to keep on top of everything, especially knowing there was something completely new coming each week. It was challenging getting used to that feeling of not finishing things and feeling okay to move on to the next thing.
     - question: What happened during/after graduation and where are you job-wise?
       answer: After graduation I chilled out for a week and took a bit of a break and then I started work on my CV and immediately after the job fair my CV started to get sent out. Very quickly after I graduated I was being enraged to apply.  I went on the trip to Brno to take part in a recruitment hackathon and then I had a crazy week of interviews. I then started to find out I had job offers and I took several days to decide which job to take and I then accepted my job at the Financial Times.
```
Comment (spike01) [inline data/graduates.yml]: Enraged?

Comment (spike01) [inline data/graduates.yml]: -> Encouraged

Comment (spike01) [inline data/graduates.yml]: Went on a trip to Brno

**Comment by nikeshashar** [comment]
Edited inline

**Pull request merged**